### PR TITLE
Feature/audit config

### PR DESCRIPTION
**Pull request by neelesh9795**
Added a submodule for supporting Audit Config Logs.

### CODE REVIEW

File: modules/audit_config/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+output "project" {
```
Comment (morgante) [inline modules/audit_config/outputs.tf]: Need header.

File: modules/audit_config/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  value       = var.project
+  description = "Project to add the IAM policies/bindings"
+}
+
+output "audit_log_config" {
+  value       = local.audit_log_config
+  description = "Map of log type and exempted members to be addded to service"
+}
```
Comment (morgante) [inline modules/audit_config/outputs.tf]: Need new line at end.

File: modules/audit_config/variables.tf [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+variable "audit_log_config" {
+  description = "Map of log type and exempted members to be addded to service"
+  type        = list(map(string))
+}
+
+variable "project" {
+  description = "Project to add the IAM policies/bindings"
+  default     = ""
```
Comment (morgante) [inline modules/audit_config/variables.tf]: We should never have invalid defaults. If a project is required, don't set a default.

File: modules/audit_config/variables.tf [hunk 4]
```diff
@@ -0,0 +1,16 @@
+variable "service" {
+  description = "Service which will be enabled for audit logging" 
+  type        = string
+  default     = "allServices"
+}
+
+variable "audit_log_config" {
+  description = "Map of log type and exempted members to be addded to service"
+  type        = list(map(string))
```
Comment (morgante) [inline modules/audit_config/variables.tf]: Could this be a list of objects?

```suggestion
  type        = list(object({log_type: string, exempted_members: list(string)))
```

File: modules/projects_iam/variables.tf [hunk 5]
```diff
@@ -35,3 +35,9 @@ variable "bindings" {
   description = "Map of role (key) and list of members (value) to add the IAM policies/bindings"
   type        = map(list(string))
 }
+
+variable "bindings_num" {
```
Comment (morgante) [inline modules/projects_iam/variables.tf]: Let's not make unrelated changes.

**Comment by neelesh9795** [comment]
Closing this PR. Opened [82](url)
